pageserver/src/metrics.rs

```diff
@@ -354,6 +383,8 @@ const STORAGE_IO_TIME_BUCKETS: &[f64] = &[
     0.001000, // 1000 usec
     0.030,    // 30 ms
     1.000,    // 1000 ms
+    5.000,    // 5000 ms
+    10.000,   // 10000 ms
```

Comment:
The comment "Roughly logarithmic scale" which was already not true even before your change. I think we used to have more entries but deleted some of them. Let's delete the comment.

Also see https://github.com/neondatabase/neon/issues/2848 and https://github.com/neondatabase/neon/pull/3850 @lassizci can we afford more granularity here?

Reply:
As a last resort, we could make the granularity configurable, and use a finer one when doing perf tests, while keeping it as is on staging/prod

Reply:
Now I only add the 30s bucket to keep the logarithmic scale.

Reply:
Adding buckets to existing metrics renders the existing ones potentially quite useless, so needs to be taken in to account. Histograms, especially with timeline dimensions are very expensive, so should be used sparingly. Then if the sample size is small, they tend to be quite inaccurate too, in which case higher level summary would perhaps work better.

That being said, we can probably afford these.